router/v3/qall.go

```diff
@@ -0,0 +1,470 @@
+package v3
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/gofrs/uuid"
+	"github.com/labstack/echo/v4"
+	"github.com/livekit/protocol/auth"
+	"github.com/livekit/protocol/livekit"
+	"github.com/livekit/protocol/webhook"
+	lksdk "github.com/livekit/server-sdk-go/v2"
+	"github.com/traPtitech/traQ/model"
+	"github.com/traPtitech/traQ/router/extension/herror"
+	"github.com/traPtitech/traQ/service/qall"
+	"go.uber.org/zap"
+)
+
+// GetQallEndpoints
```

Comment:
他のファイルであれば、メソッド名だけでなくエンドポイントや HTTP メソッドも書かれていそうなので、そちらに合わせて ↓ みたいな表記にするほうが良いかと思います
```suggestion
// GetQallEndpoints GET /qall/endpoints
```

Reply:
こちらですが、同じファイル内の他のメソッドにも適用をお願いします！